ui/address/accountHistory/AddressAccountHistoryTableItem.tsx

```diff
@@ -16,16 +16,22 @@ type Props = {
 
 const AddressAccountHistoryTableItem = (props: Props) => {
 
+  const parsedDescription = useMemo(() => {
+    const description = props.tx.classificationData.description;
+
+    return description.endsWith('.') ? description.substring(0, description.length - 1) : description;
+  }, [ props.tx.classificationData.description ]);
+
   return (
     <Tr>
-      <Td px={ 3 } py="18px" fontSize="sm" borderColor="gray.200">
+      <Td px={ 3 } py="18px" fontSize="sm" borderColor="gray.200" _dark={{ borderColor: 'whiteAlpha.200' }}>
```

Comment:
I believe the standard styles for the table should be used, you don't have to redefine them.